WordPress/gutenberg
# Footnotes: Add aria-label to return links

## What?


Adds `aria-label` to the front-end footnotes return links.

## Why?


Links with icons are not accessible.

## How?


Adds the `aria-label` attribute.

## Testing Instructions





1. Insert a footnotes block.
2. Create a footnote from a paragraph block by selecting More and then Footnote in the block toolbar.
3. Save.
4. View the front-end.
5. Notice how the link has `aria-label="Back to footnote reference X"` for the number.

### Testing Instructions for Keyboard


## Screenshots or screencast
